app/src/main/java/org/oppia/android/app/utility/FontScaleConfigurationUtil.kt

```diff
@@ -21,6 +21,7 @@ class FontScaleConfigurationUtil @Inject constructor() {
     // TODO(#3616): Migrate to the proper SDK 30+ APIs.
     @Suppress("DEPRECATION") // The code is correct for targeted versions of Android.
     windowManager!!.defaultDisplay.getMetrics(metrics)
+    @Suppress("DEPRECATION")
```

Comment:
Can you please file an issue and add a TODO here similar to #5405? It can be much simpler than that issue since there's just one code location that needs to be fixed.

Reply:
Done, filed the issue #5625.

Reply:
Perfect, thanks!